tests/test_dbos.py

```diff
@@ -1507,7 +1508,14 @@ def blocked_workflow() -> None:
             with SetWorkflowID(wfid):
                 blocked_workflow()
         assert assert_current_dbos_context().workflow_deadline_epoch_ms is None
+        start_time = time.time() * 1000
         handle = DBOS.start_workflow(blocked_workflow)
+        status = handle.get_status()
+        assert status.workflow_timeout_ms == 100
+        assert (
+            status.workflow_deadline_epoch_ms is not None
+            and status.workflow_deadline_epoch_ms > start_time
```

Comment:
Shouldn't this be > start_time + 100?

Reply:
I've tried that, it gets flaky over precision issues (failing by 0.001 ms and such)